engine/ingame/view.py

```diff
@@ -30,6 +32,7 @@ def __init__(
     ):
         self.game_world = game_world
         self.gui = gui
+        self.camera_position = [0, 0]
```

Comment:
Let's remove this for two reasons:
1) Only one function uses this value, and it doesn't need to remember it across updates. In that case a local variable would be preferable. You typically want to minimize the amount of shared mutable state that a program has.
2) The camera tracks its own state, so there's no need for it to be tracked at this level. By doing so, it violates [DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself).